docs(Java): Remove file listings: they are error prone and outdated (#1009)
cap-js/docs: java/getting-started.md

```txt
<PROJECT-ROOT>/
├─ db/
└─ srv/
   ├─ src/main/java/
   ├─ src/gen/java/
   └─ node_modules/
```

The generated folders have the following content:

| Folder | Description |
| --- | --- |
| *db* | Contains content related to your database. A simple CDS domain model is included. |
 | *srv* | Contains the CDS service definitions and Java back-end code and the sample service model. |

> | *srv/src/main/java* | Contains Java application logic. It contains a Maven project. |

Reviewer: Please change this to "Contains the Java source code of the `srv` Maven project.

Author: ```suggestion
| *srv/src/main/java* | Contains Java application logic. Contains the Java source code of the `srv/` Maven project. |
```

Indeed, this is better. :)

> | *srv/src/main/java* | Contains Java application logic. Contains the Java source code of the `srv/` Maven project. |

Author: ```suggestion
| *srv/src/main/java* | Contains the Java source code of the `srv/` Maven project. |
```

```txt
<PROJECT-ROOT>/
├─ db/

> │  └─ …

Reviewer: maybe a place holder like `<your db related cds files>`

Author: What about just leaving it empty? I think that may be cleaner. :)

```suggestion
```

└─ srv/

>    ├─ …

Reviewer: maybe a place holder like `<your db related cds files>`

Author: ```suggestion
```

   ├─ src/main/java/
   ├─ src/gen/java/
   └─ node_modules/
```

The generated folders have the following content:

| Folder | Description |
| --- | --- |
| *db* | Contains content related to your database. A simple CDS domain model is included. |
 | *srv* | Contains the CDS service definitions and Java back-end code and the sample service model. |

> | *srv/src/main/java* | Contains Java application logic. |

Author: ```suggestion
| *srv/src/main/java* | Contains Java application logic. It contains a Maven project. |
```

> | *srv/src/gen/java* | Contains the compiled CDS model and generated [accessor interfaces for typed access](./cds-data#typed-access) after building the project once. |

Reviewer: After building the project once with `mvn compile`. Because building it with IDE build support (e.g. CMD+F9 in IntelliJ) would not generate the accessor interfaces.

Author: ```suggestion
| *srv/src/gen/java* | Contains the compiled CDS model and generated [accessor interfaces for typed access](./cds-data#typed-access) after building the project with `mvn compile` once. |
```
